app/assets/templates/conversation_recipient_tag_tpl.jst.hbs

```diff
@@ -0,0 +1,12 @@
+<div class="conversation-recipient-tag clearfix btn btn-primary" data-diaspora-handle="{{ handle }}">
```

Comment:
This is not a button so I don't like using the `btn` class here. Changing the color on hover and showing the cursor as a pointer gives me the impression that you can click on it to change something. I'd rather just have the same background as the primary buttons.